docs/Server.md

```diff
@@ -812,6 +812,43 @@ The input `schema` can access all the shared schemas added with [`.addSchema`](#
 #### schemaErrorFormatter
 This property can be used set a function to format errors that happen while the `validationCompiler` fails to validate the schema. See [#error-handling](Validation-and-Serialization.md#schemaerrorformatter).
 
+<a name="schema-controller"></a>
+#### schemaController
+This property can be used to fully manage where the schemas of your application will be stored.
+It can be useful when your schemas are stored in another data structure that is unknown to fastify.
+Checkout [this issue](https://github.com/fastify/fastify/issues/2446) for an example.
+
+```js
+const fastify = Fastify({
+  schemaController: {
+    bucket: function factory (parentSchemas) {
```

Comment:
What is this factory supposed to do with `parentSchemas`? Is it an array of schema objects? A hash of validator objects?

Reply:
It is written below:

```
// It may receive as input the schemas of the parent context if some schemas has been added.
// For reference, the parentSchemas will be the returned object of the `getSchemas()` function below.
```

I will move this comment up as jsdoc 👍